indico/modules/events/registration/models/registrations.py

```diff
@@ -550,6 +551,16 @@ def visibility(self):
             return RegistrationVisibility.nobody
         return self.visibility_before_override
 
+    @property
+    def accompanying_persons(self):
+        from indico.modules.events.registration.models.form_fields import RegistrationFormFieldData
+        query = (RegistrationData.query
+                 .with_parent(self)
+                 .join(RegistrationFormFieldData)
+                 .filter(RegistrationFormFieldData.field.has(input_type='accompanying_persons')))
+        return list(itertools.chain.from_iterable(d.data for d in query.all()
+                                                  if d.field_data.field.is_active))
```

Comment:
I don't think this is correct. When you disable a field it doesn't remove it from existing registrations.

Reply:
I changed it to `not is_deleted`